content/word-lists/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Overview"
+---
+
+# Language recommendation lists 
+
+The Inclusive Naming Initiative produces 3 tiers of language recommendation. These are compiled into lists for easy consumption by adopters. Terms are separated into lists based on the severity of the term in question as well as the level of review the terms have received.
```

Comment:
3 -> three

Reply:
What @richsalz said. Most style guides I've seen, both UK & US English, instruct use of words for numbers under 10, digits for higher. This applies to all the document.